app/api/groups/update_group_require_strengthened.feature

```diff
@@ -103,18 +103,20 @@ Feature:
 
   Scenario: Should reject all pending requests when approval_change_action = 'empty'
     Given I am @Teacher
+    And the time now is "2020-01-01T01:00:00.001Z"
+    And the DB time now is "2020-01-01 01:00:00.001"
     And there are the following groups:
       | group   | parent       | members                       | require_watch_approval |
       | @School |              | @Teacher                      |                        |
       | @Class  | @ClassParent | @Student1,@Student2           | false                  |
       | @Other  |              | @Student3,@Student4,@Student5 |                        |
-    And there are the following group pending requests:
-      | group  | member    | type          |
-      | @Class | @Student1 | leave_request |
-      | @Class | @Student3 | join_request  |
-      | @Class | @Student4 | join_request  |
-      | @Class | @Student5 | invitation    |
-      | @Other | @Student5 | join_request  |
+    And the database has the following table 'group_pending_requests':
+      | group_id | member_id | type          | at                      |
+      | @Class   | @Student1 | leave_request | 2020-01-01 00:00:01.000 |
+      | @Class   | @Student3 | join_request  | 2020-01-01 00:00:03.000 |
+      | @Class   | @Student4 | join_request  | 2020-01-01 00:00:04.000 |
+      | @Class   | @Student5 | invitation    | 2020-01-01 00:00:05.000 |
+      | @Other   | @Student5 | join_request  | 2020-01-01 00:00:15.000 |
```

Comment:
having milliseconds everywhere pollutes the readability of tests.
As it is `.000`. And then if you get 6 decimals, you'll put '.000000' everywhere?
Seriously, when it is about setting values ("the database has the following table"), it should be as when you fill the db, if you do not mention the milliseconds, it assumes it is all 0s. Of course, the JSON output, you need to use the full returned value.

Reply:
We the opposite, in the JS we skip trailing zeros. Here, in tests I enter trailing zeros just because it will be used for comparing with rows returned by the DB which doesn't skip trailing zeros.

Reply:
If we skip the trailing zeros, 'the table "*" should stay unchanged' step will not work as currently we compare values as strings.